Makefile

```diff
@@ -1,30 +1,45 @@
 SENTRY_PATH := `python -c 'import sentry; print sentry.__file__.rsplit("/", 3)[0]'`
-UNAME_S := $(shell uname -s)
+PIP := python -m pip
+PIP_VERSION := 19.2.3
+PIP_OPTS := --no-use-pep517 --disable-pip-version-check
 
-develop: setup-git install-yarn
-	pip install "pip>=9,<10"
-	pip install -e git+https://github.com/getsentry/sentry.git#egg=sentry[dev,optional]
-	pip install -e ".[tests]"
-	yarn install
+develop: install-develop-requirements
 
-install-yarn:
-	@echo "--> Installing Node dependencies"
-	@hash yarn 2> /dev/null || npm install -g yarn
-	# Use NODE_ENV=development so that yarn installs both dependencies + devDependencies
-	NODE_ENV=development yarn install --ignore-optional
-
-install-tests: develop
-	pip install .[tests]
+install-pip:
+	$(PIP) install "pip==$(PIP_VERSION)"
 
 setup-git:
 	@echo "--> Installing git hooks"
 	git config branch.autosetuprebase always
 	git config core.ignorecase false
 	cd .git/hooks && ln -sf ../../config/hooks/* ./
-	pip install "pre-commit==1.18.2" $(PIP_OPTS)
+	$(PIP) install $(PIP_OPTS) "pre-commit==1.18.2"
 	pre-commit install --install-hooks
 	@echo ""
 
+install-yarn:
+	@echo "--> Installing Node dependencies"
+	@hash yarn 2> /dev/null || npm install -g yarn
+
+install-requirements: install-pip install-yarn
+	yarn install
+	# sentry optional might be unnecessary?
+	$(PIP) install $(PIP_OPTS) 'git+https://github.com/getsentry/sentry.git#egg=sentry[optional]'
```

Comment:
Will this also compile with webpack? Do you need that?

Reply:
I'm not sure, don't really know anything about how js deps with plugins work. Will circle back to this though.

Reply:
dang mark, you predicted the future https://github.com/getsentry/sentry-plugins/pull/518/commits/c36c198feab2876d22113277c23d31b0b7e55db5